libs/qa/metrics.py

```diff
@@ -68,13 +68,6 @@ class ICUBedCovidUsageTS(QAMetric):
     threshold = 10
 
 
-class ICUBedTypicalUsageRateTS(QAMetric):
-    name = "ICUBedTypicalUsageRateTS"
-    projection_path = None
-    actual_path = ["actualsTimeseries", "ICUBeds", "typicalUsageRate"]
-    threshold = 10
```

Comment:
TBH I think this whole file is basically dead code (we had some automated QA infrastructure that never really caught on), but I updated it anyway.